packages/office-ui-fabric-react/src/components/Announced/Announced.ts

```diff
@@ -0,0 +1,6 @@
+import { styled } from '../../Utilities';
+import { IAnnouncedProps, IAnnouncedStyles } from './Announced.types';
+import { AnnouncedBase } from './Announced.base';
+import { getStyles } from './Announced.styles';
+
+export const Announced = styled<IAnnouncedProps, {}, IAnnouncedStyles>(AnnouncedBase, getStyles, undefined);
```

Comment:
I believe the third argument can be omitted completely as it is _optional_.

```suggestion
export const Announced = styled<IAnnouncedProps, {}, IAnnouncedStyles>(AnnouncedBase, getStyles);
```